internal/daemon/controller/handlers/scopes/scope_service.go

```diff
@@ -81,6 +84,7 @@ var (
 			resource.Group:           groups.CollectionActions,
 			resource.HostCatalog:     host_catalogs.CollectionActions,
 			resource.Role:            roles.CollectionActions,
+			resource.Scope:           CollectionActions[2:], // Only Scope key actions are allowed on the project level
```

Comment:
Need a santity check that this is right. Not slicing the slice would make `Create` scope and `List` scope available on the project level, which I assume is why it wasn't included before.